<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MpKeyboard</class>
 <widget class="QWidget" name="MpKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SoftKeyboard</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="softKeyLayout">
    <item row="3" column="5">
     <widget class="QPushButton" name="keyFlip">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="key8">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="keyTriplet">
      <property name="text">
       <string>(3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="key2">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="keyLeft">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="key1">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QPushButton" name="keyDown">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="keyRest">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="keyNatural">
      <property name="text">
       <string>na</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="key16">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="keySharp">
      <property name="text">
       <string>sh</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="key4">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="keyDot">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="keySlur">
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="keyCmd">
      <property name="text">
       <string>C</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="keyFlat">
      <property name="text">
       <string>fl</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="key64">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="key32">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="keyShift">
      <property name="text">
       <string>S</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="noteEntry">
      <property name="text">
       <string>N</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="key128">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QPushButton" name="keyDel">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="keyTie">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="keyIns">
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QPushButton" name="keyUp">
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10">
     <widget class="QPushButton" name="keyRight">
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="keyC">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="keyD">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="keyE">
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="keyF">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="keyG">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="keyA">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="keyB">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
